joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -338,7 +225,10 @@ TEST_P(TrajectoryControllerTestParameterized, cleanup)
 
   state = traj_controller_->get_node()->cleanup();
   ASSERT_EQ(State::PRIMARY_STATE_UNCONFIGURED, state.id());
+
   // update for 0.25 seconds
+  // TODO(anyone): should the controller even allow calling update() when it is not active?
+  // update loop receives a new msg and updates accordingly
   updateController(rclcpp::Duration::from_seconds(0.25));
```

Comment:
This should be clearly removed. This is something that it doesn't happen in the real world using CM - so we should not test it at all.